share/jupyterhub/templates/login.html

```diff
@@ -68,6 +68,18 @@
         <i class="fa fa-spinner"></i>
       </div>
     </div>
+
+    {% block login_terms %}
+    {% if login_term_url %}
+    <div id="login_terms" class="login_terms">
+    <input type="checkbox" id="login_terms_checkbox" name="login_terms_checkbox"/>
```

Comment:
I believe you can use the [required attribute](https://www.w3schools.com/tags/att_input_required.asp) to avoid the need for the javascript enable/disable toggle.

Reply:
Oh great !  Didn't know about the `required` attribute.  Will test it out if it does disable/enable the submit button as we toogle the checkbox then it can completely replace the javascript, which is much better.

Thinking more about this, is there any docs I should update to expose this optional feature?

Reply:
@minrk I've implemented your suggestion.

Overall nicer since no dependency on javascript and user also get a pop-up reminder they need to check the checkbox.  

Similar pop-up behavior for both Mozilla Firefox and Google Chrome.

See animated gif:

![recorded](https://user-images.githubusercontent.com/11966697/100246404-18115e00-2f07-11eb-9061-d35434ace3aa.gif)

Reply:
Nice, thanks!